src/components/molecules/SfSidebar/SfSidebar.spec.ts

```diff
@@ -1,9 +1,9 @@
 import { shallowMount } from "@vue/test-utils";
-import SfComponent from "@/components/template/SfComponent.vue";
+import SfSidebar from "./SfSidebar.vue";
 
 describe("SfSidebar.vue", () => {
   it("renders a component", () => {
-    const component = shallowMount(SfComponent);
+    const component = shallowMount(SfSidebar);
     expect(component.contains(".sf-sidebar")).toBe(true);
```

Comment:
Imo would be good to also check `<aside>` tag, but actually the component is rendering a div, then it would also have to be changed.
What do you thing?

Reply:
good idea ;)